packages/client/foreman_ygg_worker/foreman_ygg_worker.spec

```diff
@@ -44,6 +41,9 @@ BuildRequires: go
 %else
 BuildRequires: golang
 %endif
+%if 0%{?rhel} > 8 || 0%{?fedora}
+BuildRequires: go-rpm-macros
+%endif
 
 # Use rich dependencies if available
 %if 0%{?rhel} >= 8 || 0%{?fedora}
```

Comment:
I debated if this was always available. At first I wrote a comment you can but now I realized there's also suse. We build it on 15 and I think that has them too. Have you tried that?

Reply:
Let's see how it goes

Reply:
Looks like it is supported on everything we currently build for

Reply:
I think ATIX builds on SLES 12 as well so perhaps that's an issue, but I'll leave it for them to comment on.